### PR TITLE
GOV-564: Script to add sub batch ID column in transfers table

### PR DESCRIPTION
**Pull request by arkadasfynarfin**
## Description
[GOV-425](https://mifosforge.jira.com/browse/GOV-425)

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Design related bullet points or design document link related to this PR added in the description above. 

- [ ] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [ ] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

[GOV-425]: https://mifosforge.jira.com/browse/GOV-425?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by arkadasfynarfin** [comment]
@fynmanoj Please review and merge this PR.
CC: @avikganguly01

File: .circleci/config.yml [hunk 1]
```diff
@@ -2,7 +2,7 @@ version: 2.1
 executors:
   docker-executor:
     docker:
-      - image: circleci/openjdk:17-buster-node-browsers-legacy
+      - image: circleci/openjdk:8-buster-node-browsers-legacy
```
Comment (fynmanoj) [inline .circleci/config.yml:5]: This is already fixed upstream , please rebase

**Review by fynmanoj — APPROVED** [review]
LGTM, please rebase

**Review by fynmanoj — APPROVED** [review]
need to change the migration version

**Pull request merged**